scripts/generate_code_owner_mappings.py

```diff
@@ -0,0 +1,89 @@
+"""
```

Comment:
Can we move this script to be within the new `monitoring` app?  I'd prefer placing files in edx-platform so they are co-located with their features/usage.

Reply:
Done.